openssh/files/sshd_config

```diff
@@ -69,7 +68,7 @@
 
 # Authentication:
 {{ option_default_uncommented('LoginGraceTime', 120) }}
-{{ option_default_uncommented('PermitRootLogin', 'no') }}
+{{ option_default_uncommented('PermitRootLogin', 'yes') }}
```

Comment:
This is the default setting in vanilla ssh and in pretty much most major distros. It should be up to the users to move from the default setting.